python/cuml/common/logger.pyx

```diff
@@ -73,6 +77,15 @@ level_critical = CUML_LEVEL_CRITICAL
 """Disables all log messages"""
 level_off = CUML_LEVEL_OFF
 
+cdef void _log_callback(int lvl, const char * msg) with gil:
+    print(msg.decode('utf-8'), end='')
+
+cdef void _log_flush() with gil:
+    sys.stdout.flush()
+
+Logger.get().setCallback(_log_callback)
```

Comment:
Maybe this should live at the end of the root __init__.py? I worry that it is getting initialized at a sort of arbitrary point in loading right now that might be confusing (e.g. half of load messages going through the new hook and half not)

Reply:
Fixed in latest version.